Integrations/integration-PhishMe.yml

```diff
@@ -262,25 +351,54 @@ script:
         var md = "## PhishMe IP Reputation for: "+ip+"\n";
         var dbotScore = 0;
         var threatArray = [];
-        if (threats.length !== 0) {
+        var threshold = params.ipThreshold;
+        var threshold_score = calcScore(threshold);
+        var indicatorFnd = false;
+        if (threats.length && threats.length !== 0) {
             ec.IP = {
                       Data: ip,
                       Malicious: {Vendor: 'PhishMe', Description: 'Match found in PhishMe database'}
                     };
             for (var k = 0; k < threats.length; k++) {
-                for (var i = 0; i < threats[0].blockSet.length; i++){
-                    if (threats[0].blockSet[i]['ipDetail'] && threats[0].blockSet[i]['ipDetail']['ip']===ip) {
-                        var severityLevel = threats[0].blockSet[i]['impact'];
-                        break;
+                var severityLevel = 0;
+                for (var i = 0; i < threats[k].blockSet.length; i++){
+                    if (threats[k].blockSet[i].data === ip && threats[k].blockSet[i].ipDetail) {
+                        ec.IP.ASN = threats[k].blockSet[i].ipDetail.asn;
+                        ec.IP.GEO = {
+                            "Location": (threats[k].blockSet[i].ipDetail.latitude + ', ' + threats[k].blockSet[i].ipDetail.longitude),
+                            "Country": threats[k].blockSet[i].ipDetail.countryIsoCode
+                        };
+                    }
+                    if (threats[k].blockSet[i]['impact']) {
+                        var threat_score = calcScore(threats[k].blockSet[i]['impact']);
+                        var adjusted_score = threshold_score <= threat_score ? 3 : threat_score;
+                        // if the queried indicator has a severity level, we'll take it
+                        if (threats[k].blockSet[i]['ipDetail'] && threats[k].blockSet[i]['ipDetail']['ip']===ip) {
```

Comment:
and same hire with multi block list with the indicator

Reply:
You mean the one about the indicator appearing in more than one block set?
The logic here has the same result - line 378 and 389 make sure that only the first indicator will affect the dbot score.